src/components/ingredientsForm/IngredientsForm.js

```diff
@@ -0,0 +1,49 @@
+import {useState} from 'react';
+import styled from 'styled-components';
+
+export default function IngredientsForm({onAddIngredients}) {
+  function handleInput(event) {
+    event.preventDefault();
+    const form = event.target;
+
+    updateIngredient('test');
+
+    const ingredientName = form.elements.ingredient.value.trim();
+
+    if (ingredientName.length > 0) {
+      onAddIngredients(ingredientName);
+    }
+    form.reset();
+    form.focus();
+  }
+
+  const [currentIngredient, updateIngredient] = useState('');
+
+  return (
+    <Form>
+      <Label htmlFor="ingredientInput">Add ingredients:</Label>
+      <input
+        maxLength={50}
+        value={currentIngredient}
+        onChange={handleInput}
+        id="ingredientInput"
+        name="ingredient"
+        placeholder="300g Toufu"
+      />
+      <button>
+        <span>+</span>
+      </button>
+    </Form>
+  );
+}
```

Comment:
I would probably try something like this:
```suggestion
export default function IngredientsForm({ onAddIngredient }) {
  const [currentIngredient, setCurrentIngredient] = useState('');
  
  function handleChange(event) {
    const ingredientName = event.target.value.trim();
    setCurrentIngredient(ingredientName);
  }
  
  functition addNewIngredient() {
    onAddIngredient(currentIngredient);
    setCurrentIngredient('');
  }  

  return (
    <>
      <Label htmlFor="ingredientInput">Add ingredients:</Label>
      <input
        maxLength={50}
        value={currentIngredient}
        onChange={handleChange}
        id="ingredientInput"
        name="ingredient"
        placeholder="300g Toufu"
      />
      <button onClick={addNewIngredient}>
        <span>+</span>
      </button>
    </>
  );
}
```
Maybe this will work :)